### PR TITLE
minor doc edits and docker-compose

### PR DESCRIPTION
**Pull request by eiddor**
minor doc edits
specify optional parameters in docker-compose.yaml as optional

### CODE REVIEW

File: docs/src/SUMMARY.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 - [Installation](./getting_started.md)
   - [Ubuntu/Debian](./install/debian.md)
   - [MacOS](./install/macos.md)
-  - [Docker](./install/docker.md)
+  - [Docker (Recommended)](./install/docker.md)
   - [Source](./install/source.md)
```
Comment (wouterdebie) [inline docs/src/SUMMARY.md:8]: Not sure if docker itself is recommended.. by you sure :) I'm impartial ;)

Comment (eiddor) [reply docs/src/SUMMARY.md:8]: tried to sneak that one by you haha - I'll remove it

**Comment by wouterdebie** [comment]
LGTM! Mind squashing your commits?

**Comment by eiddor** [comment]
Done!

**Pull request merged**